front/src/reducers/osrdconf/osrdConfCommon/index.ts

```diff
@@ -97,6 +102,20 @@ interface CommonConfReducers<S extends OsrdConfState> extends InfraStateReducers
   ['upsertViaFromSuggestedOP']: CaseReducer<S, PayloadAction<SuggestedOP>>;
   ['updateRollingStockComfortV2']: CaseReducer<S, PayloadAction<S['rollingStockComfortV2']>>;
   ['updateStartTime']: CaseReducer<S, PayloadAction<S['startTime']>>;
+  ['updateOriginV2']: CaseReducer<S, PayloadAction<ArrayElement<S['pathSteps']>>>;
+  ['updateDestinationV2']: CaseReducer<S, PayloadAction<ArrayElement<S['pathSteps']>>>;
+  ['updateOriginArrival']: CaseReducer<S, PayloadAction<string | undefined>>;
+  ['updateDestinationArrival']: CaseReducer<S, PayloadAction<string | undefined>>;
+  ['updateOriginArrivalType']: CaseReducer<S, PayloadAction<ArrivalTimeTypes>>;
+  ['updateDestinationArrivalType']: CaseReducer<S, PayloadAction<ArrivalTimeTypes>>;
+  ['updateOriginTolerances']: CaseReducer<
+    S,
+    PayloadAction<{ toleranceBefore: number; toleranceAfter: number }>
+  >;
+  ['updateDestinationTolerances']: CaseReducer<
+    S,
+    PayloadAction<{ toleranceBefore: number; toleranceAfter: number }>
+  >;
```

Comment:
These actions are only used in stdcm, so you can move them to `stdcmConf/index.ts` ;)